colophon.php

```diff
@@ -64,7 +65,7 @@ function team51_credits( $args = array() ) {
 						'utm_source'   => 'Automattic',
 						'utm_medium'   => 'rpc',
 						'utm_campaign' => 'Concierge Referral',
-						'utm_term'     => 'concierge',
+						'utm_term'     => $partner_domain,
```

Comment:
This should be fine if we'd like to change it -- I'd only previously left it as `concierge` as that was the prior value, for consistency before/after.

Data on UTM params: https://en.wikipedia.org/wiki/UTM_parameters